pixeltable/share/publish.py

```diff
@@ -65,6 +64,21 @@ def publish_snapshot(dest_tbl_uri: str, src_tbl: pxt.Table) -> str:
     Env.get().console_logger.info(f'The published snapshot is now available at: {confirmed_tbl_uri}')
     return confirmed_tbl_uri
 
+def get_snapshot(dest_tbl_uri: str) -> str:
+    headers_json = {'X-api-key': Env.get().pxt_api_key, 'Content-Type': 'application/json'}
+    get_request_json = {
+        'operation_type': 'get_snapshot',
+        'table_uri': dest_tbl_uri
+    }
+    request_json = {'request': get_request_json, 'operation_type': 'get_snapshot'}
```

Comment:
trying to figure out redundancy with operation_type that is used as discriminator in pydantic